### DISCUSSION TITLE
Home Assistant integration and pypi

### DISCUSSION THREAD

**Discussion by super-qua** [TomBursch/kitchenowl#491] [Ideas]
Hi 👋 
I wrote a python module and a home assistant integration for kitchenowl to be used as a shopping list. it's been sitting around now for a while, waiting for me to finish writing the tests. but I have been using the integration in my home on a daily basis and it works well.

I wonder if you'd be interested in taking it over once I wrapped it up, as I won't find the time to maintain it constantly.

**Comment by TomBursch** [comment]
Wow, that is awesome! I started working on one too, but never had the time to get it into a working state. Do you have a repository for it?

**Comment by super-qua** [comment]
yes I do. they are currently set to private though - I have to get them in shape before publishing. my developer skills are a bit rusty, and additionally it's the first time developing python and a home assistant integration 😄 
I'll ping you here once I published them. hopefully I'll get to it on the weekend.

**Comment by super-qua** [comment]
it's taking me a bit longer than expected to properly set up the project structure and write the tests. I'll keep you posted.

**Reply by TomBursch** [reply]
No worries, if you need help, please let me know!

**Reply by super-qua** [reply]
thanks, it's coming along well. did a complete refactoring this weekend and added some convenience setup stuff.
will test it in the upcoming week, but then it should be ready I think

**Comment by super-qua** [comment]
@TomBursch here are the two repositories:

- https://github.com/super-qua/ha_kitchenowl
- https://github.com/super-qua/kitchenowl-python

I created two new ones to have a fresh commit history.

As I mentioned, I do not really have the capacity to maintain them well going forward, and I'd be happy if you take them over and fork them (might also be easier for submitting to pypi or hacs). 

But of course I'd be happy to contribute then to the repositories.

**Reply by TomBursch** [reply]
Wow, love it! I'm currently on vacation but will fork&publish them when I'm back. So excited to add this to my own HA setup 😄 Thanks again for the work you did!

**Reply by super-qua** [reply]
no worries. gave me a good "excuse" to get into python and HA development :)
enjoy your vacation and let me know if you need any help setting it up
